src/services/dragonchain-client/DragonchainClient.ts

```diff
@@ -508,8 +509,8 @@ export class DragonchainClient {
   /**
    * @hidden
    */
-  private async makeRequest (path: string, method: SupportedHTTP, body: string = '', jsonParse: boolean = true) {
-    const fetchData = this.getFetchOptions(method, path, body)
+  private async makeRequest (path: string, method: SupportedHTTP, callbackURL?: string, body: string = '', jsonParse: boolean = true) {
+    const fetchData = this.getFetchOptions(method, path, body, callbackURL)
```

Comment:
Total nit-pick:
Can we even out the `getFetchOptions` arguments to match the order of `makeRequest`? This looks confusing, and took me a sec.
Suggest we just re-arrange
```suggestion
    const fetchData = this.getFetchOptions(path, method, callbackUrl, body)